packages/vertica-nodejs/mochatest/integration/client/vertica-connection-params-tests.js

```diff
@@ -106,33 +106,35 @@ describe('vertica backup_server_node connection parameter', function() {
 })
 
 describe('vertica-nodejs handling auditing connection properties', function() {
-  it('are provided automatically when establishing a connection', function() {
+  it('are provided automatically when establishing a connection', function(done) {
     const client = new vertica.Client()
     client.connect()
     client.query("SELECT client_pid, client_type, client_version, client_os, client_os_user_name, client_os_hostname FROM CURRENT_SESSION", (err, res) => {
-      if (err) assert(false)
+      if (err) done(err)
       assert.equal(res.rows[0].client_pid, process.pid)
       assert.equal(res.rows[0].client_type, "Node.js Driver")
       assert.equal(res.rows[0].client_version, vertica.version)
       assert.equal(res.rows[0].client_os, os.platform())
       assert.equal(res.rows[0].client_os_user_name, os.userInfo().username)
       assert.equal(res.rows[0].client_os_hostname, os.hostname())
       client.end()
+      done()
```

Comment:
I would expect that the explicit call to `done(err)` would fix that, but the extra call to `done()` on line 121 is probably unnecessary since the callback is provided on line 109 to the it function. . I can test it out when I play around with oauth.